test/test_regression_cli.py

```diff
@@ -109,7 +116,11 @@ def test_csv_regression(dataset, expected, tmpdir):
 
 @pytest.mark.parametrize("dataset, expected", download_md5)
 def test_download_regression(dataset, expected):
-    """Check for regression for a particular dataset downloaded only"""
+    """Check for regression for a particular dataset downloaded only
+
+    move the dataset script downloaded to the .retriever file before download
```

Comment:
> Either cd .. to move from current directorytest into the retriever main directory or to copy the scripts in to the .retriever directory.

My initial thought was to not cd into `test` in the first place. We are only doing this because the data for DelMoral and AvianBodySize is stored in `test/raw_data`. If as part of `setup_module` we copy `./test/raw_data` to `./raw_data` the data should be accessible. So we would remove the `os.chdir(os.path.dirname(os.path.abspath(__file__)))` line and replace it with something a system call that does the `raw_data` copying. The `teardown_module` looks like it's probably handling the cleanup of that already.

I'm happy to go with whatever you want, so if you like the current implementation better just let me know.